Source/WebCore/platform/Length.h

```diff
@@ -37,6 +37,7 @@ namespace WebCore {
 
 enum class LengthType : uint8_t {
     Auto,
+    Normal,
```

Comment:
@smfr @alanbaradlay @anttijk @darinadler Do you know if it's OK to add this new LengthType?